src/components/Navbar/LoginButton.tsx

```diff
@@ -55,72 +55,81 @@ export default function LoginIcon() {
     return (
         <>
             <MHidden width="smUp">
-                <MenuItem>
-                    <LoadAccountMenu type="user" />
-                </MenuItem>
-                <MenuItem>
-                    <LoadAccountMenu type="kyc" />
-                </MenuItem>
-                <MenuItem
-                    onClick={logout}
-                    sx={{ display: 'flex', justifyContent: 'space-between' }}
-                >
-                    Logout
-                    <IconButton>
-                        <Icon path={mdiLogout} size={0.8} />
-                    </IconButton>
-                </MenuItem>
+                <MenuList sx={{ backgroundColor: 'transparent' }}>
+                    <MenuItem>
+                        <LoadAccountMenu type="user" />
+                    </MenuItem>
+                    <MenuItem>
+                        <LoadAccountMenu type="alias" />
+                    </MenuItem>
+                    <MenuItem>
+                        <LoadAccountMenu type="kyc" />
+                    </MenuItem>
+                    <MenuItem
+                        onClick={logout}
+                        sx={{ display: 'flex', justifyContent: 'space-between' }}
+                    >
+                        Logout
+                        <IconButton>
+                            <Icon path={mdiLogout} size={0.8} />
+                        </IconButton>
+                    </MenuItem>
+                </MenuList>
             </MHidden>
             <MHidden width="smDown">
-                {cAddress && (
-                    <Select
-                        value={!account ? cAddress : <LoadAccountMenu type="" />}
-                        renderValue={() =>
-                            account ? <LoadAccountMenu type="" /> : `0x${cAddress}`
-                        }
-                        sx={{
-                            maxWidth: '13rem',
-                            '.MuiOutlinedInput-notchedOutline': { border: 'none' },
-                            '.MuiSvgIcon-root': { color: theme.palette.text.primary },
-                        }}
-                        onKeyDown={e => {
-                            handleKeyDown(e)
-                        }}
-                    >
-                        <MenuItem
-                            onKeyDown={e => {
-                                handleKeyDown(e)
-                            }}
-                            sx={{ typography: 'body1', width: '100%', maxWidth: '326px' }}
-                        >
-                            <LoadAccountMenu type="user" />
-                        </MenuItem>
-                        <MenuItem
-                            onKeyDown={e => {
-                                handleKeyDown(e)
+                <>
+                    <LoadAccountMenu type="alias" />
```

Comment:
Is this component only showed when user is logged in? That should be the behavior here, just want to confirm

Reply:
Yes